setup.cfg

```diff
@@ -95,7 +95,7 @@ community =
   ansible>=2.10  # GPLv3+
 # this will move to latest stable ansible as soon that is released
 core =
-  ansible-base>=2.10  # GPLv3+
+  ansible-core>=2.11.4  # GPLv3+
```

Comment:
Note that this will break envs of people who upgrade their envs with `pip install -U ansible-lint[core]` since upgrading from `ansible-base` to `ansible-core` in the pre-existing env is broken.

Reply:
I can live with that. Upgrade is not really a supported path and ansible itself have decent error messages about that. We could mark this change as behavior change in order to increase linter minor version when we merge it.

Reply:
+ maybe document it in the changelog?

Reply:
That is automatically done at https://github.com/ansible-community/ansible-lint/releases once we merge it. I made it "enhancement" which makes it bump the second version number instead of the 3rd.

Reply:
No, I mean document it properly. Just dumping the current PR title in a changelog is pointless because it doesn't contain any useful information wrt what the effect is for the end users and what to do with it.

Reply:
I modified that PR description to mention implications and reasoning. All changelog entries do have links to the PRs. I also ensured that it will be included in 5.2.0 due to potentially breaking change.

This release/changelog was used unchanged for more than two years (since v4.3.0) and changing it is would clearly be outside the scope of this review. I hope that is addressing current concerns.